UCSCore/src/test/java/it/cnr/iit/ucscore/coverage/ContextHandlerCoverageTests.java

```diff
@@ -41,28 +40,25 @@
 @DirtiesContext( classMode = ClassMode.BEFORE_EACH_TEST_METHOD )
 @EnableAutoConfiguration
 @ComponentScan( basePackages = { "it.cnr.iit" } )
-@ContextConfiguration( classes = { UCSRestProperties.class, TestProperties.class } )
+@ContextConfiguration( classes = { TestProperties.class } )
 @RunWith( SpringRunner.class )
 public class ContextHandlerCoverageTests extends UCSRestBaseTests {
 
-    @Autowired
-    private UCSProperties properties;
-
     private String policy;
     private String request;
 
     @PostConstruct
     private void init() throws URISyntaxException, IOException, JAXBException {
         log.info( "Init tests " );
         Thread.interrupted(); // Avoid a nasty exception
-        policy = FileUtility.readFileAsString( testProperties.getPolicyFile() );
-        request = FileUtility.readFileAsString( testProperties.getRequestFile() );
+        policy = FileUtility.readFileAbsPath( testProperties.getPolicyFile() );
```

Comment:
why do we no longer need readFileAsString?
we should change the readFileAbsPath to be renamed as AsString

Reply:
readFileAsString was not working properly in all cases.
ok I can rename that to AsString

Reply:
done

Reply:
thnx